### PR TITLE
[bugfix] generate the 'worker_cpu_affinity' config

### PR DESCRIPTION
**Pull request by Miss-you**
fix issue #1657 
generate the 'worker_cpu_affinity' config

NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

**Comment by Miss-you** [comment]
@membphis

File: bin/apisix [hunk 1]
```diff
@@ -21,6 +21,8 @@ local function trim(s)
     return (s:gsub("^%s*(.-)%s*$", "%1"))
 end
 
+-- Note: The `excute_cmd` return value will have a line break at the end,
+-- it is recommended to use the `trim` function to handle the return value.
```
Comment (membphis) [inline bin/apisix:25]: we can move this comment to https://github.com/apache/incubator-apisix/pull/1658/files#diff-62edadffc237f13dc28a694080d293a8R670

**Comment by membphis** [comment]
@Miss-you please rebase your branch with `master`.  This branch has conflicts that must be resolved.

File: bin/apisix (hunk 1 above)
Comment (Miss-you) [reply bin/apisix:25]: I think it's reasonable to put this function usage note in the corresponding function definition.

**Comment by Miss-you** [comment]
> @Miss-you please rebase your branch with `master`. This branch has conflicts that must be resolved.

Okay, I'll resolve the conflict tonight.

File: bin/apisix (hunk 1 above)
Comment (membphis) [reply bin/apisix:25]: ok

**Comment by membphis** [comment]
> Okay, I'll resolve the conflict tonight.

I did it for you.

**Comment by Miss-you** [comment]
> > Okay, I'll resolve the conflict tonight.
> 
> I did it for you.

thank you

**Comment by membphis** [comment]
merged, many thx @Miss-you

**Pull request merged**